OasisDEX/oasis-borrow
# [sc-11316] [sc-11317] Extends Ajna borrow with close feature

# [Close and Adjust action on ajna borrow](https://shortcutLink)
  
## Changes 👷‍♀️
- Adds cose and adjsut functionality to ajna borrow
  
## How to test 🧪
- Open or navigate to borrow position
- The dropdown should contain two new entries: `borrow` and `adjust`
- Select `adjust`
- Form should allow you to adjust your position, both ways to increase and decrease risk, the order confirmation should contain relevant data for borrow position (it will be different to multiply)
- Test if transaction passes and the resulting position is as expected 
- Select `close` from dropdown
- Now form should allow you to close position to either collateral or debt token
- Try if transaction passes

Gotchas:
- The two additional functionalities shouldn't appear for pools where multiply is not supported, e.g. custom pools
